src/config/main-config.js

```diff
@@ -0,0 +1,19 @@
+require('dotenv').config();
+const path = require('path');
+const logger = require('morgan');
+const cors = require('cors')
+const bodyParser = require('body-parser');
+
+module.exports = {
+  init(app, express) {
+    app.use(bodyParser.urlencoded({ extended: true }));
+    app.use(logger('dev'));
+    app.use(express.json());
+    app.use(cors());
```

Comment:
I think it's okay to leave this here, but it's probably not necessary since I doubt we'll be serving across origins.

Reply:
I just used it in case. Not sure if this will affect anything negatively anyways.